zen/scene/scene.c

```diff
@@ -51,6 +51,66 @@ zn_scene_move_board_binding_handler(
   }
 }
 
+static void
+zn_scene_switch_focused_view_binding_handler(
+    uint32_t time_msec, uint32_t key, void* data)
+{
+  struct zn_scene* scene = data;
+  UNUSED(time_msec);
+
+  if (scene->focused_view == NULL) return;
+  // Should be mapped
+  struct zn_board* focused_board = scene->focused_view->board;
+  if (focused_board == NULL) {
+    zn_error("Something went wrong");
+    return;
+  }
+  if (wl_list_length(&focused_board->view_list) == 1) {
+    zn_warn("Only 1 view");
+    return;
+  }
+
+  struct zn_view* new_focused_view;
+  if (key != KEY_TAB) {
+    zn_error("Invalid key: %u", key);
+    return;
+  }
+  // bring prev to front (should be the second last)
+  // put the current focused_view to the last
+  new_focused_view = zn_container_of(
+      focused_board->view_list.prev->prev, new_focused_view, link);
+  struct zn_view* original_focused_view = scene->focused_view;
+  zn_scene_set_focused_view(scene, new_focused_view);
+  zn_view_bring_to_back(original_focused_view);
+}
+
+static void
+zn_scene_align_focused_view_binding_handler(
+    uint32_t time_msec, uint32_t key, void* data)
+{
+  struct zn_scene* scene = data;
+  UNUSED(time_msec);
+
+  if (scene->focused_view == NULL) return;
+  struct zn_board* board = scene->focused_view->board;
```

Comment:
We have a suitable function.
The same is applied to `zn_scene_switch_focused_view_binding_handler`
```suggestion
  struct zn_board* board = zn_scene_get_board(scene);
  if (board == NULL) { ... }
```